manifest_staging/charts/csi-secrets-store-provider-azure/values.yaml

```diff
@@ -32,6 +37,10 @@ linux:
       maxUnavailable: 1
 
 windows:
+  image:
+    repository: mcr.microsoft.com/oss/azure/secrets-store/provider-azure
```

Comment:
How come this image/repo is identical to Linux, is there some magic in the MCR repo that knows when its windows or when it's linux ?

We are having issue where the image gets pulled / cached on our proxy and depending which node last pulled the image (windows vs linux) the cached image becomes compatible with only that version of the OS